build.gradle

```diff
@@ -30,7 +30,7 @@ tasks.withType(JavaCompile).configureEach {
 minecraft {
     mcVersion = '1.12.2'
 
-    username = "Desoroxxx"
+    username = "Dev"
```

Comment:
Why, dev doesn't even have a skin

Reply:
/ Please change it to your username but don't push as change

Reply:
> / Please change it to your username but don't push as change

That's not my username...

Reply:
> Why, dev doesn't even have a skin

Great question you'd think for a name like that he'd at least have a skin

Reply:
> > / Please change it to your username but don't push as change
> 
> That's not my username...

Yeah, that's the problem, you should set this as your username if you care about it, but don't push that, or just leave it like that.